scripts/tests/test_zkai_d128_attention_mlp_boundary_frontier_gate.py

```
@@ -0,0 +1,93 @@
+        cases = payload["mutation_result"]["cases"]
+        self.assertEqual(payload["mutation_inventory"]["cases"], list(gate.MUTATION_NAMES))
+        self.assertEqual(len(cases), len(gate.MUTATION_NAMES))
+        self.assertTrue(all(case["rejected"] for case in cases))
+
+    def test_promoting_single_native_status_rejects(self) -> None:
+        payload = self.fresh_payload()
+        payload["routes"]["single_native_attention_plus_derived_mlp_fused"][
+            "status"
+        ] = "GO_NATIVE_ATTENTION_PLUS_MLP_PROOF_OBJECT_EXISTS"
+        gate._refresh_route_and_payload(payload)
+        with self.assertRaisesRegex(gate.AttentionMlpBoundaryFrontierError, "route drift"):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)